src/components/common/buttons/IconButton.js

```diff
@@ -33,7 +33,6 @@ const IconButton = ({ styles, theme, text, onPress, disabled, name, style, ...ic
     >
       <CustomIcon
         color={theme.colors.darkBlue}
-        name={name}
```

Comment:
why was removed ?

Reply:
it was add to iconProps

Reply:
it's still used in CustomIcon line 21 https://github.com/GoodDollar/GoodDAPP/pull/3837/files/9400f4a76d6db8595e6f479394698db87cf395b6#diff-dc6d03882bdbdafd4c68e05a21e2d3224c1c9dff31813071c98f0f050a7b746eR21

Reply:
It is still passed to CustomIcon, we have rest props, named iconProps, which contains name and others